### ISSUE TITLE
add helicone options

### ISSUE THREAD

**Issue by hwchase17** [langchain-ai/langchainjs#121]
something like:

```
const model = new OpenAI(
    {},
    {
       ...HeliconeOptions({
          propertySession: "24",
          propertyConversation: "support_issue_2",
          propertyApp: "mobile",
          cacheEnabled: false
       }),
       otherAxiosConfigurationOption: "value"
    }
)
```

**Comment by spark-ming** [comment]
@hwchase17 I think this can be closed now that #120 gives guidance on how to achieve this.

Probably no need to a custom HeliconeOptions interface.

**Comment by spark-ming** [comment]
Not sure how important this is, given you can achieve all Helicone functionality with this anyway #120
